src/Application.php

```diff
@@ -170,4 +173,32 @@ public static function setEnvironment(): void
             throw new ErrorException($errstr, 0, $errno, $errfile, $errline);
         });
     }
+
+    protected function checkUserPermissions(): void
+    {
+        $realUser = getenv('KBC_REALUSER');
+        $dbUsername = $this->config->getData()['parameters']['db']['user'] ?? null;
```

Comment:
Toto by som dal do osobitnej `protected function getDbUserName()` metody, ak by v nejakom extractore bol uzivatel ulozeny pod inym klucom. ... a mozno rovno aj `getRealUserName()` ak by to bolo niekde potrebne spravit trocha inac.